assets/scss/_registry.scss

```diff
@@ -39,3 +39,47 @@
 .registry-entry {
   @extend .shadow;
 }
+
+// Optimize styles and responsiveness for dark and light mode, see https://getbootstrap.com/docs/5.3/customize/color-modes/
+#searchForm .btn.btn-outline-success,
+.btn-outline-danger,
+.btn-outline-secondary {
+  &:hover {
+    color: var(--bs-white);
+  }
+}
+
+.badge.rounded-pill.text-bg-primary {
+  font-weight: $font-weight-bold;
+}
+
+$color-mode-type: data;
+// Define dark mode colors for data container
+$dark-border-color: darken(#cccccc, 20%);
+$background-color: map-get($otel-colors, 'dark-background');
```

Comment:
what is the expected effect of those variables? I assume they have a global effect, so either we consider making them part of `_variables_project.scss`

Reply:
The variables are supposed to make the shadow drop effect for the elements to be visible in dark mode